cli/http_cache.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
Add rust doc comment for module which references relevant RFC

Reply:
You might also want to call out some ways in which this first pass implementation is not RFC compliant in TODO comments.

Reply:
added a TODO